code/drasil-code/Language/Drasil/Choices.hs

```diff
@@ -67,10 +68,20 @@ data Choices = Choices {
   auxFiles :: [AuxFile]
 }
 
+class RenderChoices a where
```

Comment:
So, in theory, the class `RenderChoices` should not really exist. All of the 'data' store in instances of `RenderChoices` should instead be stored in chunks. However, I will not ask you to change that right now - but please make an issue for a future refactor.